extension/main.js

```diff
@@ -86,7 +86,11 @@
 			ghTab.url = url + 'notifications';
 		}
 
-		if (tab.url === '' || tab.url === 'chrome://newtab/' || tab.url === ghTab.url) {
+		if (window.GitHubNotify.settings.get('useParticipatingCount')) {
+			ghTab.url += '/participating';
+		}
+
+		if (typeof tab !== 'undefined' && (tab.url === '' || tab.url === 'chrome://newtab/' || tab.url === ghTab.url)) {
```

Comment:
In the case of dev console I believe I ran into tab not being set.
I can remove if you wish. I've found it generally a good idea to test the existence of each level of an object before accessing subkeys.

Shall I remove it?